Ally4/Your-Vote
# #171686868 Signup controller

#### What does this PR do?

- `Helps user signup in the platform`

#### Description of Task to be completed?

- `Create controller file`

- `Create router file`

- `Create model file`

- `Create server`

- `Create validation file`

- `Create dotenv`

- `Create babel file`

- `Installation of libraries`

- `Installation of eslint`

#### How should this be manually tested?

- `By cloning the repository and installing the required libraries, run the app and check using postman`

#### Any background context you want to provide?

- `Update of eslint`

#### What are the relevant pivotal tracker stories?

- `https://www.pivotaltracker.com/story/show/171686868`

#### Screenshots (if appropriate)
![Screenshot (3310)](https://user-images.githubusercontent.com/44581262/76234346-47295780-6232-11ea-9472-23b07c0c7f7c.png)
![Screenshot (3312)](https://user-images.githubusercontent.com/44581262/76234349-498bb180-6232-11ea-8237-edbf6718ad0d.png)
![Screenshot (3306)](https://user-images.githubusercontent.com/44581262/76234336-4264a380-6232-11ea-8e39-7264637eb706.png)
![Screenshot (3305)](https://user-images.githubusercontent.com/44581262/76234329-40024980-6232-11ea-8afe-dde6bd7ae2b3.png)
![Screenshot (3311)](https://user-images.githubusercontent.com/44581262/76234348-485a8480-6232-11ea-8ca1-53cb2f398498.png)
![Screenshot (3307)](https://user-images.githubusercontent.com/44581262/76234340-442e6700-6232-11ea-811f-6ace0dcd5333.png)
![Screenshot (3308)](https://user-images.githubusercontent.com/44581262/76234341-455f9400-6232-11ea-91f8-7b3d7bb50bc3.png)
![Screenshot (3309)](https://user-images.githubusercontent.com/44581262/76234345-45f82a80-6232-11ea-96ce-216117df8250.png)
